docs/content/preview/yugabyte-platform/install-yugabyte-platform/prerequisites/default.md

```diff
@@ -33,7 +33,7 @@ You can install YugabyteDB Anywhere on the following Linux distributions:
 
 - CentOS (default)
 - Ubuntu 18 and 20, via Replicated
-- Other [operating systems supported by Replicated](https://www.replicated.com/docs/distributing-an-application/supported-operating-systems/).
+Refer also to the other [requirements for installation via Replicated](https://docs.replicated.com/enterprise/installing-general-requirements).
```

Comment:
the list of OSs has since been updated, drop these changes and get what's on master